eng/.docsettings.yml

```diff
@@ -21,42 +21,97 @@ required_readme_sections:
+  - ['sdk/applicationinsights/Microsoft.Azure.ApplicationInsights.Query/README.md','#5499']
+  - ['sdk/containerregistry/Microsoft.Azure.ContainerRegistry/README.md','#5499']
+  - ['sdk/cognitiveservices/Language.LUIS.Runtime/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingEntitySearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingWebSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Knowledge.QnAMaker/README.md','#5499']
+  - ['sdk/cognitiveservices/AnomalyDetector/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingVisualSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingCustomSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingImageSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Vision.FormRecognizer/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingVideoSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingNewsSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Personalizer/README.md','#5499']
+  - ['sdk/cognitiveservices/Vision.ComputerVision/README.md','#5499']
+  - ['sdk/cognitiveservices/Vision.CustomVision.Training/README.md','#5499']
+  - ['sdk/cognitiveservices/Vision.Face/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingAutoSuggest/README.md','#5499']
+  - ['sdk/cognitiveservices/Language.LUIS.Authoring/README.md','#5499']
+  - ['sdk/cognitiveservices/Vision.ContentModerator/README.md','#5499']
+  - ['sdk/cognitiveservices/Vision.CustomVision.Prediction/README.md','#5499']
+  - ['sdk/cognitiveservices/Language.SpellCheck/README.md','#5499']
+  - ['sdk/cognitiveservices/Language.TextAnalytics/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingLocalSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/Search.BingCustomImageSearch/README.md','#5499']
+  - ['sdk/cognitiveservices/FormRecognizer/README.md','#5499']
+  - ['sdk/batch/Microsoft.Azure.Batch.FileStaging/README.md','#5499']
+  - ['sdk/graphrbac/Microsoft.Azure.Graph.RBAC/README.md','#5499']
+  - ['sdk/batch/Microsoft.Azure.Batch/Tools/README.md','#5499']
+  - ['sdk/keyvault/Microsoft.Azure.KeyVault.Core/README.md','#5499']
+  - ['sdk/eventhub/Microsoft.Azure.EventHubs.Processor/README.md','#5499']
+  - ['sdk/keyvault/Microsoft.Azure.KeyVault.Cryptography/README.md','#5499']
+  - ['sdk/search/Microsoft.Azure.Search/README.md','#5499']
+  - ['sdk/keyvault/Microsoft.Azure.KeyVault.WebKey/README.md','#5499']
+  - ['sdk/search/Microsoft.Azure.Search.Data/README.md','#5499']
+  - ['sdk/batch/Microsoft.Azure.Batch/Tools/README.md','#5499']
+  - ['sdk/search/Microsoft.Azure.Search.Service/README.md','#5499']
+  - ['sdk/keyvault/Microsoft.Azure.KeyVault.Extensions/README.md','#5499']
+  - ['sdk/search/Microsoft.Azure.Search.Common/README.md','#5499']
+  - ['sdk/eventhub/Microsoft.Azure.EventHubs.ServiceFabricProcessor/README.md','#5499']
+  # List for changelogs begins here
```

Comment:
Can we just add `*\CHANGELOG.md` to the omitted paths section up top instead of having to update this for every new changelog?

Reply:
But then we'd have to explicitly include them as they are fixed. IMO, I like this way of explicitly tracking them along with the bug reference to fix them individually (as issue tasks).

Reply:
Well that will apply to all changelog.mds' which is not really what we want. We still want, to verify presence of changelogs just not Track 1 changelogs

Reply:
I agree with @heaths that we should avoid the wildcards here for things that we know would like to fix. If we use a wildcard we might as well not have the validation happening at all.

Reply:
Oh, I looked up #5499 and assumed based on the bug being used that we wanted to block all changelogs.  That's fine, but I'd file a new bug then.